src/modals/vm-manage-gpu/ManageGpuModal.js

```diff
@@ -65,8 +76,12 @@ const ManageGpuModal = ({
 
     const allSelectedGpus = gpus.filter(gpu => gpu.requestedInstances > 0)
     const d = displayOn_ === undefined ? displayOn : displayOn_
+    let p
+    if (compatibilityVersion >= CompatibilityVersion.VERSION_4_7) {
+      p = driverParams_ === undefined ? driverParams : driverParams_
```

Comment:
Will it silently delete driverParams if present and version is < 4.7?  This can happen e.g. when a VM has custom compatibility level set and later unset, without noticing.

If this is the case then it would be better to keep the parameters and perhaps indicate some hint that they may not be applied.  Note that driver parameters are sent to the host in the backend even when version is 4.7 and they may be applied or ignored there, depending on the Vdsm version running on the host.  So it is OK to not allow defining the parameters on < 4.7 but it may not be very user friendly just to drop them silently if they are already defined.

Reply:
> Note that driver parameters are sent to the host in the backend even when version is 4.7 and they may be applied or ignored there, depending on the Vdsm version running on the host. So it is OK to not allow defining the parameters on < 4.7 but it may not be very user friendly just to drop them silently if they are already defined.

If I understand this part correctly then that's actually an argument for clearing it - so it won't be sent when the compatibility level is changed back to < 4.7. but that's really not that interesting case, let's not hold this PR on that

Reply:
I think we should not handle the driver params for VMs < 4.7 at all. I've changed the patch so that the values are not displayed nor stored if the VM has CL < 4.7.